src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -1,4 +1,49 @@
 package nl.novi.TechItEasy.controllers;
 
+import org.apache.coyote.Response;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
Elke Mapping begint bij jou met "/televisions". In plaats van dat je dat bij elke mapping opnieuw schrijft, kun je dat beter hiet in de RequestMapping defineren. Dit zou dan dus worden "/api/televisions"

Op die manier hoef je het ook maar op 1 plak aan te passen als je besluit om inplaats van "televisions" toch "bananen" te verkopen